test/pom.xml

```diff
@@ -14,7 +14,6 @@
 	<properties>
 		<m2eclipse.wtp.contextRoot>iaf-test</m2eclipse.wtp.contextRoot>
 		<failOnMissingWebXml>false</failOnMissingWebXml>
-		<databaseHost>localhost</databaseHost>
 	</properties>
```

Comment:
Deze moet hier niet weg, dan kunnen we iaf-test niet meer los draaien

Reply:
Dan moet ik hem weghalen uit de iaf/pom.xml, anders is hij dubbel.

Reply:
Gedaan.